cni/azure-windows.conflist

```diff
@@ -40,6 +40,46 @@
                         "DestinationPrefix": "10.0.0.0/8",
                         "NeedEncap": true
                     }
+                },
+                {
```

Comment:
is there any reason you chose to add these ACLs by default in the CNI conflist?

Reply:
This is Azure CNI - The ACL is needed because of the security incident that needs to block access to wireserver API ( Which exist in Azure). My line of thinking is that considering it is security related, every container should get this default rather the partner unknowingly enabling it and creating a security hole for them.